geomesa-kafka/geomesa-kafka-tools/bin/install-kafka.sh

```diff
@@ -7,43 +7,39 @@
 # http://www.opensource.org/licenses/apache2.0.php.
 #
 
-kafka_major_version="0.8"
+kafka_major_version="%%kafka.maj.version%%"
 
 if [[ (-z "$1") ]]; then
   echo "Error: Provide one arg which is the target directory (e.g. /opt/jboss/standalone/deployments/geoserver.war/WEB-INF/lib)"
-  echo "You may also use 0.9 as an optional second parameter if using kafka 0.9"
   exit
 fi
-if [[ (! -z "$2") ]]; then
-  kafka_major_version=$2
-fi
+
 install_dir=$1
 NL=$'\n'
 read -r -p "Install Kafka ${kafka_major_version} DataStore dependencies to ${install_dir}?${NL}Confirm? [Y/n]" confirm
 confirm=${confirm,,} #lowercasing
 if [[ $confirm =~ ^(yes|y) || $confirm == "" ]]; then
   # get stuff
-  if [[ $kafka_major_version = "0.8" ]]; then
-    declare -a urls=(
-      "https://search.maven.org/remotecontent?filepath=org/apache/kafka/kafka-clients/${kafka.version}/kafka-clients-${kafka.version}.jar"
-      "https://search.maven.org/remotecontent?filepath=org/apache/kafka/kafka_2.11/${kafka.version}/kafka_2.11-${kafka.version}.jar"
-      "https://search.maven.org/remotecontent?filepath=com/yammer/metrics/metrics-core/2.2.0/metrics-core-2.2.0.jar"
-      "https://search.maven.org/remotecontent?filepath=com/101tec/zkclient/0.3/zkclient-0.3.jar"
-      "https://search.maven.org/remotecontent?filepath=org/apache/zookeeper/zookeeper/${zookeeper.version}/zookeeper-${zookeeper.version}.jar"
-      )
-  elif [[ $kafka_major_version = "0.9" ]]; then
-    declare -a urls=(
-      "https://search.maven.org/remotecontent?filepath=org/apache/kafka/kafka-clients/0.9.0.1/kafka-clients-0.9.0.1.jar"
-      "https://search.maven.org/remotecontent?filepath=org/apache/kafka/kafka_2.11/0.9.0.1/kafka_2.11-0.9.0.1.jar"
-      "https://search.maven.org/remotecontent?filepath=com/yammer/metrics/metrics-core/2.2.0/metrics-core-2.2.0.jar"
-      "https://search.maven.org/remotecontent?filepath=com/101tec/zkclient/0.7/zkclient-0.7.jar"
-      "https://search.maven.org/remotecontent?filepath=org/apache/zookeeper/zookeeper/${zookeeper.version}/zookeeper-${zookeeper.version}.jar"
-      )
+
+  if [[ $kafka_major_version = "08" ]]; then
+    ZK_CLIENT_VERSION="0.3"
+  elif [[ $kafka_major_version = "09" ]]; then
+    ZK_CLIENT_VERSION="0.7"
+  elif [[ $kafka_major_version = "10" ]]; then
+    ZK_CLIENT_VERSION="0.8"
   else
-    echo "Error: Kafka version ${kafka_major_version} is not supported.  Please select 0.9 as second arg or omit to use default (0.8)."
+    echo "Error: Kafka version ${kafka_major_version} is not supported."
     exit
   fi
 
+  declare -a urls=(
+    "https://search.maven.org/remotecontent?filepath=org/apache/kafka/kafka-clients/%%kafka.min.version%%/kafka-clients-%%kafka.min.version%%.jar"
```

Comment:
would you mind pulling out the URL up top like I think we do in one of the other scripts? helps for when we have to install it in places with no internets.